app/admin/events.rb

```diff
@@ -0,0 +1,51 @@
+ActiveAdmin.register Event do
+  menu label: 'All Events', priority: 3, parent: 'Administration'
```

Comment:
question: how to keep only `index` action for CSV download, but without adding the 'events' page anywhere?

Reply:
You can add `menu: false`